src/interceptors/fetch.js

```diff
@@ -4,6 +4,13 @@ import parseUrl from 'parse-url';
 
 const nativeFetch = window.fetch;
 
+//TODO: Handle response headers
+let fakeResponse = function(response = {}) {
+  const responseStr = JSON.stringify(response);
+
+  return new Response(responseStr);
```

Comment:
Yes, it must need to be a string because if not later when the user do `response.json()` it will fail (since it can't convert to json something that is already a json)